class/defaults.yml

```
@@ -35,6 +35,8 @@ parameters:
       extraEnv: {}
       resourceRatio:
         memoryPerCore: 4Gi
+      nodeSelector: {}
+      tolerations: []
```

While you're at it, I'd make scheduling the agent on the master nodes the default:

```suggestion
      nodeSelector:
        node-role.kubernetes.io/master: ""
      tolerations:
        - effect: "NoSchedule"
          key: "node-role.kubernetes.io/master"
          operator: "Exists"
```

---

I thought about that, but that would make the component openshift specific which it is designed not to be.